olive/passes/onnx/conversion.py

```diff
@@ -392,6 +392,9 @@ def _convert_model_on_device(
         """Convert a PyTorchModelHandler to an ONNXModelHandler."""
         # load the model
         pytorch_model, model_attributes = self._load_pytorch_model(model, device, torch_dtype)
+        if is_peft_model(pytorch_model):
```

Comment:
We don't want this to always be done. The llama qlora example depends on the adapaters not being merged. 

Can you make this an option?

Reply:
Done.